Servers/types/framework.type.ts

```
@@ -0,0 +1,17 @@
+import { Transaction } from "sequelize";
+import { createEUFrameworkQuery, deleteProjectFrameworkEUQuery } from "../utils/eu.utils";
+import { createISOFrameworkQuery, deleteProjectFrameworkISOQuery } from "../utils/iso42001.utils";
+
+export const frameworkAdditionMap: Record<number, (
+  projectId: number,
+  enable_ai_data_insertion: boolean,
+  transaction: Transaction
+) => Promise<Object>> = {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)